addons/l10n_cl/models/account_move.py

```diff
@@ -24,32 +24,32 @@ def _get_document_type_sequence(self):
 
     def _get_l10n_latam_documents_domain(self):
         self.ensure_one()
-        domain = super()._get_l10n_latam_documents_domain()
-        if (self.journal_id.l10n_latam_use_documents and
-                self.journal_id.company_id.country_id == self.env.ref('base.cl')):
-            if self.journal_id.type == 'sale':
-                document_type_ids = self.journal_id.l10n_cl_sequence_ids.mapped('l10n_latam_document_type_id').ids
-            else:
-                partner_domain = [
-                    ('country_id.code', '=', 'CL'),
-                    ('internal_type', 'in', ['invoice', 'debit_note', 'credit_note', 'invoice_in'])]
-                if not self.partner_id:
-                    pass
-                elif self.partner_id.l10n_cl_sii_taxpayer_type == '1' and self.partner_id_vat != '60805000-0':
-                    partner_domain += [('code', 'not in', ['39', '70', '71', '914', '911'])]
-                elif self.partner_id.l10n_cl_sii_taxpayer_type == '1' and self.partner_id_vat == '60805000-0':
-                    partner_domain += [('code', 'not in', ['39', '70', '71'])]
-                elif self.partner_id.l10n_cl_sii_taxpayer_type == '2':
-                    partner_domain += [('code', 'in', ['70', '71', '56', '61'])]
-                elif self.partner_id.l10n_cl_sii_taxpayer_type == '3':
-                    partner_domain += [('code', 'in', ['35', '38', '39', '41', '56', '61'])]
-                elif not self.partner_id.l10n_cl_sii_taxpayer_type or self.partner_id.country_id != self.env.ref(
-                        'base.cl') or self.partner_id.l10n_cl_sii_taxpayer_type == '4':
-                    partner_domain += [('code', 'in', [])]
-                document_type_ids = self.env['l10n_latam.document.type'].search(partner_domain).ids
-            domain = expression.AND([domain, [('id', 'in', document_type_ids)]])
+        if self.journal_id.company_id.country_id != self.env.ref('base.cl') or not \
+                self.journal_id.l10n_latam_use_documents:
+            return False
```

Comment:
Sorry  @Danisan but what if you install AR and CL at the same time e.g.?  We need a super here e.g. return super()._get_latam_documents_domain()

Reply:
ok you are right..